build.sc

```diff
@@ -236,11 +236,13 @@ object dev extends MillModule{
 
     write(outputPath, prependShellScript())
 
-    val perms = java.nio.file.Files.getPosixFilePermissions(outputPath.toNIO)
-    perms.add(PosixFilePermission.GROUP_EXECUTE)
-    perms.add(PosixFilePermission.OWNER_EXECUTE)
-    perms.add(PosixFilePermission.OTHERS_EXECUTE)
-    java.nio.file.Files.setPosixFilePermissions(outputPath.toNIO, perms)
+    if (!System.getProperty("os.name").toLowerCase.startsWith("windows")) {
```

Comment:
Does windows filesystem support a similar set of group/owner/other execute permissions? Or are those concepts just non-existent on windows?

Reply:
It throws `UnsuportedOperationException` on Windows.